### ISSUE TITLE
Ligature request: !=

### ISSUE THREAD

**Issue by jpcaruana** [i-tu/Hasklig#51]
Java (and some others) has `!=`. It should look like `/= `

Thanks

**Comment by OlegKrikun** [comment]
+1, java users need the `!=`. Plz add this =)

**Comment by ZeBriD** [comment]
It would be great!

**Comment by cseelus** [comment]
Fira Code and Monoid both do a great job of displaying the not equal sign.

Fira Code:
![screen shot 2016-05-18 at 19 57 27](https://cloud.githubusercontent.com/assets/3188392/15369429/cce82598-1d32-11e6-9b81-07130a66eb57.png)


Monoid:
![screen shot 2016-05-18 at 19 56 12](https://cloud.githubusercontent.com/assets/3188392/15369362/9f2d714e-1d32-11e6-9ca9-cff04646a40b.png)

**Comment by lstatie** [comment]
I would love to see this work as well. I use Visual Studio 2015 and it is weird to see == work, but != not.
Also,  => works, but -> does not.
I really love the way Hasklig looks as I write code, but the ligatures are a bit hit and miss.

**Comment by aqeeliz** [comment]
@lstatie You can try [v1.0 beta](https://github.com/i-tu/Hasklig/releases/tag/v1.0-beta), it increases support for applications. @i-tu recommended it on another issue.

**Comment by lstatie** [comment]
I tried v1.0 beta and thought I saw the ligature for !=, but for the last few days, I do not see it any more. The same ligatures that worked before are still working, but the ones that 1.0 Beta is supposed to activate do not work. 
Any ideas as to why?

**Comment by ForNeVeR** [comment]
This is actually a duplicate of (poorly-named) #43.

**Comment by hauleth** [comment]
What is the status?

**Comment by sos4nt** [comment]
Related to #78 – the `/=` ligature is already problematic in Ruby, but making `!=` and `/=` look identical would break the font.

**Comment by Arnie97** [comment]
As a non-haskell programmer, I've just found [Black Font](https://github.com/blackro-se/blackfont) (a Hasklig fork) as a nice alternative, which provides support for `!=` instead of `/=`.

**Comment by vikky49** [comment]
Any idea when this ligature would be available..?? This is the most common thing that i use every day .Though its available in FiraCode i like this font way better than the others..

**Comment by johnknoop** [comment]
@Arnie97 Is there anyway I can download a built version of Black Font?

**Comment by nelsongonzalez** [comment]
I would like to have != ligature too, it's very comon in java and javascript.

!= and !== would be nice.

**Comment by h-2** [comment]
Yes, this is really important for C, C++, D, Java, Javascript, AWK...

**Comment by gonzaloserrano** [comment]
Would love it! <3

**Comment by bradynpoulsen** [comment]
+1

**Comment by vikky49** [comment]
Any plans for supporting this ligature ? This would make the font complete

**Comment by rifcoder** [comment]
Vote for != as well, it's really must have

**Comment by orffen** [comment]
Any updates to this?

**Comment by BornForFever** [comment]
> As a non-haskell programmer, I've just found [Black Font](https://github.com/blackro-se/blackfont) (a Hasklig fork) as a nice alternative, which provides support for `!=` instead of `/=`.

Black Font is also a nice font. But why is it less popular? I cannot understand
